tests/opt-full-hess-every/input.dat

```diff
@@ -0,0 +1,20 @@
+# SCF/sto-3g optimization with a hessian every step
+
+molecule h2o {
+     O
+     H 1 1.0
+     H 1 1.0 2 104.5
+}
+
+set {
+  basis sto-3g
+  e_convergence 10
+  d_convergence 10
+  geom_maxiter 5
+  full_hess_every 1
+  g_convergence gau_verytight
```

Comment:
Any reason to use very tight here? Be good if the test could execute quickly.

Reply:
When I tried with `gau_tight`, it took exactly as many iterations: 3. I can change to `gau_tight` if you want, but it would be a purely cosmetic change unless something changes with the optimizer. And in that case, better safe than sorry.